src/debugger/handler_dbgp.c

```diff
@@ -2279,7 +2377,7 @@ static int xdebug_dbgp_parse_option(xdebug_con *context, char* line, int flags,
 #define FD_RL_SOCKET  1
 
 
-static char* xdebug_fd_read_line_delim(int socketfd, fd_buf *context, int type, unsigned char delim, int *length)
+int xdebug_fd_read_line_delim(int socketfd, fd_buf *context, int type, int block, unsigned char delim, char **cmd, int *length)
```

Comment:
You're introducing the `block` argument here, but I can't see it being used in the function.